enterprise/server/remote_execution/containers/firecracker/firecracker_test.go

```diff
@@ -504,7 +527,9 @@ func TestFirecrackerNonRoot(t *testing.T) {
 		t.Fatal(res.Error)
 	}
 	require.NoError(t, res.Error)
-	assert.Equal(t, "uid=65534(nobody) gid=65534(nobody)\n", string(res.Stdout))
+	require.Empty(t, string(res.Stderr))
+	require.Equal(t, 0, res.ExitCode)
+	require.Equal(t, "uid=65534(nobody) gid=65534(nobody)\n", string(res.Stdout))
```

Comment:
probably fine but be aware https://stackoverflow.com/questions/34831861/can-i-assume-that-nobody-is-65534